### PR TITLE
One ccl edits

### PR DESCRIPTION
**Pull request by emmwalsh**
Made edits as suggested by team

### CODE REVIEW

**Comment by emmwalsh** [comment]
@aepanchi looping you in to keep you up to date.

File: doc/rst/source/env-variables.rst [hunk 1]
```diff
@@ -68,13 +68,13 @@ CCL_ALLGATHERV_MONOLITHIC_PIPELINE_KERNEL
...
-     - Uses compute kernels to transfer data across GPUs for the ``ALLGATHERV`` collective. 
+     - Uses compute kernels to transfer data across GPUs for the ``ALLGATHERV`` collective. The default value.
    * - ``0``
-     - Uses copy engines to transfer data across GPUs for the ``ALLGATHERV`` collective. The default value.
+     - Uses copy engines to transfer data across GPUs for the ``ALLGATHERV`` collective. 
   
 **Description**
 
-Set this environment variable to enable compute kernels for the ``ALLGATHERV`` collective using device (GPU) buffers. 
+Set this environment variable to enable compute kernels that pipeline data transfers across tiles in the same GPU with data transfers across different GPUs,  for the ALLGATHERV collective using device (GPU) buffers.
```
Comment (aepanchi) [inline doc/rst/source/env-variables.rst:77]:
```suggestion
Set this environment variable to enable compute kernels that pipeline data transfers across tiles in the same GPU with data transfers across different GPUs for the ``ALLGATHERV`` collective using device (GPU) buffers. 
```

File: doc/rst/source/env-variables.rst [hunk 2]
```diff
@@ -98,13 +98,13 @@ CCL_REDUCE_SCATTER_MONOLITHIC_PIPELINE_KERNEL
...
-     - Uses compute kernels for the ``ALLREDUCE``, ``REDUCE``, and ``REDUCE_SCATTER`` collectives. 
+     - Uses compute kernels for the ``ALLREDUCE``, ``REDUCE``, and ``REDUCE_SCATTER`` collectives. The default value. 
    * - ``0``
-     - Uses copy engines to transfer data across GPUs for the ``ALLREDUCE``, ``REDUCE``, and ``REDUCE_SCATTER collectives``. The default value. 
+     - Uses copy engines to transfer data across GPUs for the ``ALLREDUCE``, ``REDUCE``, and ``REDUCE_SCATTER collectives``. 
   
 **Description**
 
-Set this environment variable to enable compute kernels, that pipeline data transfers across tiles in the same GPU and across different GPUs, for the ``ALLREDUCE``, ``REDUCE``, and ``REDUCE_SCATTER`` collectives using the device (GPU) buffers. 
+Set this environment variable to enable compute kernels that pipeline data transfers across tiles in the same GPU with data transfers across different GPUs, for the ALLREDUCE, REDUCE, and REDUCE_SCATTER collectives using the device (GPU) buffers.
```
Comment (aepanchi) [inline doc/rst/source/env-variables.rst:107]:
```suggestion
Set this environment variable to enable compute kernels that pipeline data transfers across tiles in the same GPU with data transfers across different GPUs for the ``ALLREDUCE``, ``REDUCE``, and ``REDUCE_SCATTER`` collectives using the device (GPU) buffers. 
```

File: doc/rst/source/env-variables.rst [hunk 3]
```diff
@@ -135,6 +135,35 @@ CCL_ALLTOALLV_MONOLITHIC_KERNEL
...
+   :widths: 25 50
+   :header-rows: 1
+   :align: left
+
+   * - <value>
+     - Description
+   * - ``1``
+     - Enable SYCL kernels.
+   * - ``0``
+     - Disable SYCL kernels. Default value.
```
Comment (aepanchi) [inline doc/rst/source/env-variables.rst:159]:
```suggestion
     - Disable SYCL kernels. The default value.
```

File: doc/rst/source/env-variables.rst [hunk 4]
```diff
@@ -135,6 +135,35 @@ CCL_ALLTOALLV_MONOLITHIC_KERNEL
...
+  
+**Description**
+
+Setting this environment variable to 1 enables SYCL kernel-based implementation for allgatherv, allreduce, and reduce_scatter.
```
Comment (aepanchi) [inline doc/rst/source/env-variables.rst:163]:
```suggestion
Set this environment variable to ``1`` to enable SYCL kernel-based implementation for ``ALLGATHERV``, ``ALLREDUCE``, and ``REDUCE_SCATTER``.
```

File: doc/rst/source/env-variables.rst [hunk 5]
```diff
@@ -135,6 +135,35 @@ CCL_ALLTOALLV_MONOLITHIC_KERNEL
...
+This is a new optimization that optimizes all message sizes and supports some data types (int32, fp32, fp16, and bf16), sum operation, and single node.
```
Comment (aepanchi) [inline doc/rst/source/env-variables.rst:164]:
```suggestion
This new optimization enhances all message sizes. It supports the ``int32``, ``fp32``, ``fp16``, and ``bf16`` data types, sum operations, and single nodes. 
```

File: doc/rst/source/env-variables.rst [hunk 6]
```diff
@@ -135,6 +135,35 @@ CCL_ALLTOALLV_MONOLITHIC_KERNEL
...
+   * - ``1``
+     - Enable SYCL kernels.
+   * - ``0``
+     - Disable SYCL kernels. Default value.
+  
+**Description**
+
+Setting this environment variable to 1 enables SYCL kernel-based implementation for allgatherv, allreduce, and reduce_scatter.
+This is a new optimization that optimizes all message sizes and supports some data types (int32, fp32, fp16, and bf16), sum operation, and single node. 
+oneCCL falls back to other implementations when the support is not available with SYCL kernels, so the user can safely setup this environment variable
```
Comment (aepanchi) [inline doc/rst/source/env-variables.rst:165]:
```suggestion
If the support is unavailable with SYCL kernels, oneCCL falls back to other implementations. Therefore, you can safely set up this environment variable. 
```

File: doc/rst/source/introduction/installation.rst [hunk 7]
```diff
@@ -91,3 +91,7 @@ You can customize CLI-based installation (for example, specify directory, compil
   ::
 
      make -j VERBOSE=1 install
+
+Find More
+*********
+ - `oneCCL GitHub Page <https://oneapi-src.github.io/oneCCL>`_
```
Comment (aepanchi) [inline doc/rst/source/introduction/installation.rst:97]: isn't is already merged? if so, delete it from this PR

**Pull request closed without merging**